try:
    print(10/5)
except:
    print("Error Occured!")
else:
    print("Else Part")

# else part will get executed only when try block have not raised any exception